doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -705,9 +705,10 @@ command in CoqIDE.  You can change the background colors shown for diffs from th
 lets you control other attributes of the highlights, such as the foreground
 color, bold, italic, underline and strikeout.
 
-Note: As of this writing (August 2018), Proof General will need minor changes
-to be able to show diffs correctly.  We hope it will support this feature soon.
-See https://github.com/ProofGeneral/PG/issues/381 for the current status.
+As of June 2019, Proof General can also display Coq-generated proof diffs automatically.
+Please see the PG documentation section
+"`Showing Proof Diffs" <https://proofgeneral.github.io/doc/userman/ProofGeneral_12/#Showing-Proof-Diffs>`_)
```

Comment:
```suggestion
"`Showing Proof Diffs" <https://proofgeneral.github.io/doc/userman/Coq-Proof-General/#Showing-Proof-Diffs>`_)
```

Ref.: https://github.com/ProofGeneral/PG/pull/421#issuecomment-497982497

Reply:
@jfehrle @cpitclaudel As mentioned in ProofGeneral/PG#421 I've rewritten the URL of all PG's online doc to have more stable URLs.

Reply:
Sounds good. Should we add redirects, or do we have sufficiently few links?

Reply:
Hi @cpitclaudel,
According to [this search](https://github.com/search?q=https%3A%2F%2Fproofgeneral.github.io%2Fdoc%2Fuserman%2F&type=Code) there are a few such links for `userman` in GitHub, so I've just added redirects using the [jekyll-redirect-from](https://blog.webjeda.com/jekyll-redirect/) plugin, for both `userman` and `adaptingman`.
(cf. the corresponding [commits](https://github.com/ProofGeneral/proofgeneral.github.io/compare/99779ab2ce11cff12b674dafed20ea65f01eeeff...b680be021896d9e3cb83700265ef1880654e5b55))

Reply:
Super